_lzbench/lzbench.cpp

```diff
@@ -744,6 +745,34 @@ void usage(lzbench_params_t* params)
     fprintf(stderr,"  " PROGNAME " -t0u0i3j5 -ezstd fname = the same as above with aggregated parameters\n");
 }
 
+char* cpu_brand_string(void)
+{
+    uint32_t mx[4], i, a, b, c, d;
+
+    #if (defined(__i386__) || defined(__x86_64__))
+    char* cpu_brand_str = (char*)malloc(3*sizeof(mx)+1);
```

Comment:
If not defined __i386__ or __x86_64__ or malloc failed cpu_brand_string() will return NULL. It's still gona free(line 1000) cpu_brand.

Reply:
and please use calloc instead of malloc.